diff -Naur old/via.desktop new/via.desktop
--- old/via.desktop	2020-12-22 21:29:31.000000000 +0100
+++ new/via.desktop	2021-09-11 16:23:05.532322492 +0200
@@ -3,10 +3,10 @@
 Version=1.0
 Type=Application
 Terminal=false                
-Exec=/opt/sviesolutions/via/via %u
-Icon=/opt/sviesolutions/via/viaIcone.svg
+Exec=via %u
+Icon=viaIcone.svg
 Name=Via Launcher
 Comment=Via Launcher
 Categories=Application;Network;
 MimeType=x-scheme-handler/viaapp;
-X-KDE-Protocols=viaapp;
\ No newline at end of file
+X-KDE-Protocols=viaapp;
diff -Naur old/viascreensharing.desktop new/viascreensharing.desktop
--- old/viascreensharing.desktop	2020-12-22 20:43:43.000000000 +0100
+++ new/viascreensharing.desktop	2021-09-11 16:28:38.195382403 +0200
@@ -3,10 +3,10 @@
 Version=1.0
 Type=Application
 Terminal=false                
-Exec=/opt/sviesolutions/viascreensharing/viascreensharing %u
-Icon=/opt/sviesolutions/viascreensharing/viascreensharing.svg
+Exec=viascreensharing %u
+Icon=viascreensharing.svg
 Name=ViaScreenSharing Launcher
 Comment=ViaScreenSharing Launcher   
 Categories=Application;Network;
 MimeType=x-scheme-handler/viascreensharing;
-X-KDE-Protocols=viascreensharing;
\ No newline at end of file
+X-KDE-Protocols=viascreensharing;
